src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc

```diff
@@ -896,7 +896,7 @@ TEST_F(HMICommandsNotificationsTest,
   utils::SharedPtr<Command> command =
       CreateCommand<OnDeviceStateChangedNotification>(message);
 
-  std::string device_id = "device_id";
+  std::string device_id = "AA15F2204D6B";
```

Comment:
@vlantonov , No. It couldn't, becase in next line `EXPECT_CALL(policy_interface_, RemoveDevice(_)).WillOnce(GetArg(&device_id));` the value of argument of `RemoveDevice(_)` is storing in that `device_id`